forgerock-authenticator/src/androidTest/java/org/forgerock/android/auth/MockModelBuilder.java

```diff
@@ -37,19 +37,31 @@ public static Account createAccount(String issuer, String accountName, String im
 
     public static OathMechanism createOath(String mechanismUID, String issuer, String accountName, OathMechanism.TokenType oathType,
                                            String algorithm, String secret, int digits, long counter, int period) {
-        OathMechanism oath = OathMechanism.builder()
-                .setMechanismUID(mechanismUID)
-                .setIssuer(issuer)
-                .setAccountName(accountName)
-                .setType(oathType)
-                .setAlgorithm(algorithm)
-                .setSecret(secret)
-                .setDigits(digits)
-                .setCounter(counter)
-                .setPeriod(period)
-                .build();
 
-        return oath;
+        switch (OathMechanism.TokenType.TOTP) {
```

Comment:
should be `switch (OathMechanism.TokenType.TOTP) {`?

Reply:
Good catch, updated.